tests/unit_tests/forward_model_runner/test_job.py

```diff
@@ -80,9 +80,16 @@ def max_memory_per_subprocess_layer(layers: int) -> int:
                 max_seen = max(max_seen, status.memory_status.max_rss)
         return max_seen
 
+    # size of the list that gets forked. we will use this when
+    # comparing the memory used with different amounts of forks done
+    memory_per_numbers_list = sys.getsizeof(int(0)) * 1e6
```

Comment:
could you make it into a single statement? `memory_per_numbers_list = 0.9 * (sys.getsizeof(int(0)) * 1e6)`

Reply:
if you wish. I just liked splitting the comments :)